GetDKAN/dkan-tools
# Commands to work with dkan2 and some tests.

*QA Steps*

1. Make sure you have `dktl` installed in your host
1. On the host while standing in the `dkan-tools` repo root, run `composer install` to get phpunit
1. Finally run the tests from the where you are stading with `./vendor/bin/phpunit`

_Note:_ Some commands of `dktl` require user intervention, pay attention to the tests as they run to make sure they do not get stuck. Making sure we can run the tests without this intervention would be useful. Also the tests take a while as they are getting and bunch of stuff and building it, etc.
